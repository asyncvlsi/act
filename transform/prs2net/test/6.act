defproc foo(bool a, b, c)
{
  prs {
    a & b -> c-
  }
}

foo x;
