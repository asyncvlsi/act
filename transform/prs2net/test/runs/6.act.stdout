*
*---- act defproc: foo<> -----
* raw ports:  a b c
*
.subckt foo a b c
*.PININFO a:I b:I c:O
*.POWER VDD Vdd
*.POWER GND GND
*.POWER NSUB GND
*.POWER PSUB Vdd
*
* --- node flags ---
*
* c (state-holding): pup_reff=-1; pdn_reff=0.8
*
* --- end node flags ---
*
M0_ #fb6# c Vdd Vdd pch W=0.15U L=0.06U
M1_keeper #7 GND Vdd Vdd pch W=0.12U L=0.33U
M2_ #3 a GND GND nch W=0.15U L=0.06U
M3_ #fb6# c GND GND nch W=0.15U L=0.06U
M4_keeper c #fb6# GND GND nch W=0.12U L=0.06U
M5_ c b #3 GND nch W=0.15U L=0.06U
M6_keeper c #fb6# #7 Vdd pch W=0.12U L=0.06U
.ends
*---- end of process: foo<> -----
